pegen/pegen.c

```diff
@@ -641,6 +641,17 @@ asdl_seq *seq_map_to_alias(Parser *p, asdl_seq *seq)
     return new_seq;
 }
 
+asdl_seq *names_map_to_id(Parser *p, asdl_seq *seq)
```

Comment:
I'm not sure I understand how you came with this name. I'd put the verb first and keep the plurality the same, e.g. 'map_names_to_ids'.

Also the type should go on a line by itself.

Reply:
Well, I just followed the naming I had introduced with `seq_map_to_alias`. I'll change all three of them.

Reply:
I guess I didn't object to `seq_map_to_alias` because you have a bunch of other `seq_*` functions. But now I look at it, it seems that that one should really be called `map_pegen_aliases_to_aliases`, which is a bit long... Maybe that one should be `extract_orig_aliases`? Because it seems it takes a sequence of PegenAliases (which wrap an alias_ty together with line/col info) and returns a sequence of those alias_ty values.

Reply:
Done.